src/screens/Carousel.js

```diff
@@ -99,6 +96,21 @@ export default () => (
           </PropertyValue>
         </Property>
 
+        <Property name="children">
+          <Description>
+            Array of child nodes to use for slides of the Carousel.
+          </Description>
+          <PropertyValue type="array">
+            <Example>
+              {`
+                <Box>Slide One</Box>
+                <Box>Slide Two</Box>
+                <Box>Slide Three</Box>
+              `}
```

Comment:
```suggestion
              {`[
  <Box>Slide One</Box>,
  <Box>Slide Two</Box>,
  <Box>Slide Three</Box>
]`}
```
I wondering if we should wrap with brackets here since if you were passing children as an array through a named prop (`<Carousel children={} />` vs. `<Carousel> {children} </Carousel>`) you would need to have the brackets.

Also we should get rid of the extra tab spacing. If it is left it it will show up on the docs page with the extra tabs. See other props with multiline strings for examples.